Project_dissemination/HISCO Slides/Slides.Rmd

```diff
@@ -193,31 +188,73 @@ https://youtu.be/BF_oNe-sABQ?si=Ef1YrkK43Ln_IHJS
 ]
 
 ---
-### Open questions:
+
+
+.pull-left[
+## Open questions
+- Historical developments in living standards
 - The role of technology and change of technology
 - The dynamics of income inequality 
 - Social mobility 
 - Returns to education
 
 
-### Barriers:
+## Two broad barriers to research
+- We need data
+  + A: We need to to take data from archives
+  + B: ...And turn it into something useful for analysis
+]
+
 
-.pull-left-narrow[
-**Type A:**
+.pull-right[
+**Type A:**  
 ```{r echo=FALSE, message=FALSE, warning=FALSE, out.height="300px", out.width="300px"}
 knitr::include_graphics("Figures/TypeA.png")
 ```
 
-]
-
-
-.pull-right[
 .red[**Type B:**]
 ```{r echo=FALSE, message=FALSE, warning=FALSE, out.height="300px"}
 knitr::include_graphics("Figures/TypeB.webp")
 ```
 ]
 
+---
+## Thanks to data contributors
```

Comment:
Seems at a glance 23 sources, similar to older version. Is this correct, i.e., are there any new sources we should remember to add?